elliott

```diff
@@ -748,7 +767,47 @@ Example to look for bugs that have been moved back to MODIFIED
                           comment = "Elliott changed bug status from {initial} to {final}.".format(initial=original_state, final=new_state),
                           private = True)
     
-    green_print("{count} bugs successfullly modified".format(bug_count))
+    green_print("{count} bugs successfullly modified".format(changed_bug_count))
+
+
+#
+# Search for CVE tracker bugs
+#
+@cli.command("find-cve-trackers", short_help="Find and list CVE tracker bugs forSecurity issues.")
+@click.option('--cve',
+              required=False,
+              help="CVE number to filter on. (ex. CVE-2011-1000000)")
+@click.option('--status', 'status',
+              multiple=True,
+              required=False,
+              default=['NEW', 'ASSIGNED', 'POST', 'MODIFIED', 'ON_QA', 'VERIFIED', 'RELEASE_PENDING'],
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status the bugs can be in.")
+@pass_runtime
+def find_cve_trackers(runtime, cve, status):
+    """Find Red Hat Bugzilla security bugs and list them out. Automatic attachment of these
+    bugs is not supported because security issues generally need to be hand crafted to make
+    sure all requirements are met.
+
```

Comment:
It's easy to forget this when the `--group` option isn't in the `--help` output. How do you feel about copying the note from the create command:

> You MUST specify a group (ex: "openshift-3.9") manually using the
>    --group option. See examples below.

Or reword it or something if it's not to your taste?